src/libNode/DSBlockProcessing.cpp

```diff
@@ -405,9 +405,38 @@ void Node::StartFirstTxEpoch()
         }
     }
 
+    // Choose N other DS nodes to be recipient of microblock
+    m_DSMBReceivers.clear();
+    unsigned int numOfMBReceivers = NUM_MICROBLOCK_GOSSIP_RECEIVERS;
+    if (m_mediator.m_DSCommittee->size() < numOfMBReceivers)
+    {
+        numOfMBReceivers = m_mediator.m_DSCommittee->size();
+    }
+
+    for (unsigned int i = 0; i < numOfMBReceivers; i++)
+    {
+        m_DSMBReceivers.emplace_back(m_mediator.m_DSCommittee->at(i).second);
+    }
+
+    m_consensusLeaderID = 0;
```

Comment:
Can you double-check the need for this line 421? I just remember @kaikawaliu had a PR which involved moving this line to somewhere else, if I'm not mistaken.